onnx/common/ir_pb_converter.cc

```diff
@@ -266,7 +267,27 @@ std::unique_ptr<Graph> graphProtoToGraph(
       n->setDomain(np.domain());
     }
   }
-
+  // If ir_version >= 4, initializer does not have to be included in input
+  // Add value_info from initializer
+  // Store for input recovery
+  size_t original_input_size = g->inputs().size();
+  if (ir_version >= 4) {
+    for (int i = 0; i < gp.initializer_size(); ++i) {
+      auto vip = tensorProtoToTensor(gp.initializer(i));
+      // Only update input if input does not exists
+      if (value_by_name_of.count(vip.name()) == 0) {
+        auto v = g->addInput();
+        v->setElemType(vip.elem_type());
+        std::vector<Dimension> sizes;
+        for (auto size: vip.sizes()) {
+          sizes.push_back(size);
+        }
+        v->setSizes(sizes);
+        v->setUniqueName(vip.name());
+        value_by_name_of[vip.name()] = v;
```

Comment:
Could this part be replaced by using `addInitializerAndInput` in `ir.h`?

Reply:
Good suggestion. Updated. Thank you